locations/spiders/tanger_outlet.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+class TangerOutletSpider(scrapy.Spider):
+	name = "tanger_outlet"
+	allowed_domains = ["www.tangeroutlet.com"]
+	download_delay = 0.1
+	start_urls = (
+		"https://www.tangeroutlet.com/locations",
+	)
+
+	def parse(self, response):
+		storeurls = response.xpath('.//a[@title="Visit"]/@href').extract()
+		storeurls = [response.urljoin(i) for i in storeurls if 'location' in i]
+		for storeurl in storeurls:
+			yield scrapy.Request(storeurl, callback=self.parse_store)
+
+	def parse_store(self, response):
+		tempurl = '/' + response.url.split('/')[-2] + '/'
+		for i in response.xpath('.//ul[@id="usCenters"]/li[@class="allCenters"]/a/@href').extract():
+			if i == tempurl:
+				latitude = float(response.xpath('.//ul[@id="usCenters"]/li[@class="allCenters"]/@data-longitude').extract_first())
```

Comment:
It looks like the lat and lon aren't getting parsed properly. Take a look at the output from the test run:

https://placescraper-results.s3.amazonaws.com/map.html?show=https://placescraper-results.s3.amazonaws.com/ci/284655293/tanger_outlet/output.geojson

Reply:
Oops, my bad. I switched the latitude and longitude while storing their values. Will correct that!